sklearn/utils/estimator_checks.py

```diff
@@ -56,8 +56,9 @@
 
 BOSTON = None
 CROSS_DECOMPOSITION = ['PLSCanonical', 'PLSRegression', 'CCA', 'PLSSVD']
-MULTI_OUTPUT = ['CCA', 'DecisionTreeRegressor', 'ElasticNet',
-                'ExtraTreeRegressor', 'ExtraTreesRegressor', 'GaussianProcess',
+MULTI_OUTPUT = ['CCA',  'DecisionTreeClassifier', 'DecisionTreeRegressor',
```

Comment:
In the function check_supervised_y_2d the line inside the warning `section-
``` estimator.fit(X, y[:, np.newaxis])``` does not give any warnings for the classifiers I included. therefore I included it in the MULTI_OUTPUT list . Otherwise it would give me a error that ```expected 1 DataConversionWarning, got:     ```. I checked sklearn documents for DecisionTreeClassifier it says y can accept [n_samples, n_outputs].